Get blockchain from height (#524)
witnet/witnet-rust: docs/interface/cli.md

Block for epoch #46925 had digest 2dc469691916a862154eb92473278ea8591ace910ec7ecb560797cbb91fdc01e
```

> There are two optional arguments: epoch and limit. For example, to get the

Reviewer: ```suggestion
There are two optional arguments: `epoch` and `limit`. For example, to get the
```

> blocks from epochs 100-104, use epoch 100 and limit 5:

Reviewer: ```suggestion
blocks for epochs `100-104`, use `epoch 100` and `limit 5`:
```

```sh
$ witnet cli getBlockChain -c witnet_01.toml 100 5
```

If a negative epoch is supplied, it is interpreted as "the last N epochs".

> For instance, to get the blocks from the last epoch:

Reviewer: ```suggestion
For instance, to get the block for the last epoch:
```
